action.yml

```diff
@@ -33,6 +33,12 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Check if AWS CLI is installed
+      shell: bash
+      run: |
+        echo "Checking if AWS CLI is installed..."
+        aws --version
```

Comment:
If this isn't installed would it be better to just add a step to install it or do we want the workflow to error out and stop running?

Reply:
A lot of work and logic to set up install commands based on the kind of container/image the workflow is being run on

Reply:
Makes sense, I guess it won't matter for most things as long as they're using either ubuntu-latest or our self-hosted runner.